### PR TITLE
fix: displayed toolbar for map icon click

### LINKED ISSUE #1531: Toolbar not displayed on a maps icon click in SessioDetailActivity
**Actual Behaviour**

Currently when i click on maps icon in SessionDetailActivity the toolbar is not displayed.

**Expected Behaviour**

Toolbar to be displayed on the click.

**Steps to reproduce it**
-

**LogCat for the issue**

-

**Screenshots of the issue**

18405718_10211243040586292_947593921_o

18379230_10211243040666294_733808373_o

**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

https://github.com/fossasia/open-event-android/milestone/24

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1531 

Changes: 
Toolbar displayed on map icon click in SessionDetailActivity.

Screenshots for the change: 

18379389_10211243409035503_449479925_o

### CODE REVIEW

**Comment by arp95** [comment]
@the-dagger 
please review!

**Comment by Shailesh351** [comment]
LGTM @arp95 just rebase your branch on development

**Comment by Shailesh351** [comment]
Yes, map icon is also disappearing after back button is pressed on MapFragment.
Fix it along with it

**Comment by heysadboy** [comment]
@Shailesh351 I don't think we need that map icon here as we are already on the map screen.

**Comment by Shailesh351** [comment]
@heysadboy I am saying that once back button pressed on map fragment  when user return on session detail activity map icon is disappered

**Comment by arp95** [comment]
@Shailesh351 
rebasing it.
actually i tried to ensure the map icon doesnt disappear for a day but found no solutions to it.
any ideas?

File: android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java [hunk 1]
```diff
@@ -362,10 +363,10 @@ public void onBackPressed() {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.action_map:
-                // Hide all the views except the frame layout
+                // Hide all the views except the frame layout adn toolbar
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: Just fix this spelling mistake, everything else is fine

**Comment by iamareebjamal** [comment]
@arp95 Are you adding the map icon programmatically?

**Comment by arp95** [comment]
@iamareebjamal 
have done changes.

**Comment by iamareebjamal** [comment]
@arp95 Is the menu icon still disappearing?

**Comment by arp95** [comment]
@iamareebjamal 
thats a separate issue because if you see in SessionActivity even there on clicking, the maps icon disappears. As its not realted realtedd over here will have a lookm at both map icon at a single pr instead of here.

is it fine then?

**Pull request merged**